a3fe/tests/test_run_integration.py

```diff
@@ -0,0 +1,214 @@
+"""Integration tests for a3fe.
+
+These tests are meant to be run locally before releasing code,
+and are not run on CI due to their long runtime.
+
+Note: This file contains dedicated integration tests that are more granular than
+the integration test in test_run.py (which is currently skipped).
+
+See README.md in this directory for more information on running these tests.
```

Comment:
Could you also please copy over the helpful stuff in this README.md to the docs? Thanks!